docs/quickstarts/remix.mdx

```diff
@@ -29,11 +29,13 @@ Once you have a Remix application ready, you need to install Clerk's Remix SDK.
 
 ### Set environment keys
 
-Below is an example of an `.env` file. If you are signed into your Clerk Dashboard, your keys should become visible by clicking on the eye icon.
+Below is an example of an `.env.local` file. 
```

Comment:
This is minor, but we should probably say "an example of **a** `.env.local` file", because it's pronounced "dot env dot local" when you're reading it